### ISSUE TITLE
Deploying a second jaws project to the same account fails for silly reasons

### ISSUE THREAD

**Issue by jontg** [jaws-framework/JAWS#229]
In the cloud formation spin-up, I get stuck with
```
	10:20:48 UTC-0700	CREATE_FAILED	AWS::S3::Bucket	JawsBucket	jaws.dev.uswest2.myapp.com already exists
```

In the *second* JAWS project I try to create.

**Comment by jwulf** [comment]
Perhaps because you gave it the project the same "domain"?

This is the namespace of the project - it's the `myapp.com` part of the bucket name. So you've got: `jaws` - always the same. `dev` - stage name. `uswest2` - region, and `myapp.com` - domain (namespace).

So you want to specify a different domain for your second project.

**Comment by sixlettervariables** [comment]
This confused me as well when I started, I had assumed the project name would be placed onto the S3 bucket name.

**Comment by ac360** [comment]
:+1:  S3 buckets must have globally unique names within AWS regions.  The  `jaws.dev.uswest2.myapp.com` default provided in the JAWS new project screen is going to keep running into collisions like the @jontg describes.

Further, the issue is more complex because once an S3 bucket is part of a CF stack, you can't use/reference it in another CF stack (maybe you can, not entirely sure, but still feels like a complex user experience), like we had originally hoped.

We've been trying to figure this out.  It looks like the only solution is to keep the project S3 bucket outside of CloudFormation.  I don't like this because it leaves a resource outside of your CloudFormation template, making JAWS slightly more confusing at first glance and this is the stuff that keeps me up at night.  If you have any other suggestions, please contribute!

**Comment by doapp-ryanp** [comment]
FWIW s3 buckets must be globally unique across all of AWS.  There is also a soft limit of buckets per account, that can be increased but you must submit a request and it takes time (and you must give a business justification)

**Comment by sixlettervariables** [comment]
I think the CF stack limit hits before the bucket limit (only something like 20 stacks). Would it be less confusing for something like: `{project-name}.jaws.{stage}.{region}.{domain}`? Also less likely to be in conflict.

**Comment by doapp-ryanp** [comment]
we got this solved in the upcoming release. see https://github.com/jaws-framework/JAWS/issues/270

**Comment by MarkCheshire** [comment]
:+1:
